mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -206,33 +207,42 @@ public static PartitionSpec spec(Configuration config) {
    * @param config The target configuration to store to
    * @param table The table which we want to store to the configuration
    */
+
```

Comment:
Nit: unnecessary newline? Also separates the method from Javadoc.

Reply:
Fixed the javadoc to (I suspect I have left the extra line when I started to fix the doc and got distracted :)